content/docs/user-guide/gems/reference/multiplayer/multiplayer-debug/debug/desync-audit-trail.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Multiplayer Desync Audit Trail
+description: Analyze and Debug Multiplayer Desyncs
+linkTitle: Desync Audit Trail
+---
+
+When the Client and Server disagree on the value of a networked variable, a desync occurs. Desyncs can be extremely difficult to debug which is where the Multiplayer Desync Audit Trail can help.
+
+The Multiplayer Desync Audit Trail is a tool available with the Multiplayer Gem in Debug environments. It details all network desyncs that occur and categorizes network activity leading up to a desync to attempt to root cause the desync itself.
+
+![Audit Trail Overlay](/images/user-guide/gems/reference/multiplayer/audit_trail_default.png)
+
+The Audit Trail will capture a desync including the Input ID and Host Frame it occurred on. For every desync, the UI will list all the captured activity leading to the desync. The depth of this history can be controlled by a CVar.
+
+## Configuration
+Various CVARs can be modified to enable additional data capture. Enabling additional capture both adds a performance cost and density to the Audit Trail. If you require capturing a longer event history, consider raising *net_DebutAuditTrail_HistorySize*.
+
+| Property | Description | Type |
+|---|---|---|---|
+| cl_EnableDesyncDebugging | If true, enables output of Desyncs to standard logs AND Audit Trail capture. | `bool` |
+| cl_DesyncDebugging_AuditInputs | If true, adds Inputs to audit trail. | `bool` |
+| net_DebutAuditTrail_HistorySize | Maximum number of events the Audit Trail will aggregate. Raising this value is recommended if Input or Event Auditing is enabled to compensate for increased Event volume. | `int` |
+
+## Auditing Categories
+### Desync
+
+Desyncs are the primary event captured by the Audit Trail. Consequently the Audit Trail organizes all activity around Desyncs. On the client, desyncs generally include a delta map of variables and the values the client and server disagreed on
+
+### Input
+
+Network inputs detail actions that create deltas in the networked state of the simulation. The audit trail lists all network inputs that were sent. In addition it also lists non default values for each member per network input. This allows the correlation of inputs to desynchronized data. This can help clarify what actions may have led to a desync.
+
+![Audit Trail Inputs](/images/user-guide/gems/reference/multiplayer/audit_trail_input.png)
+
+The above screenshot shows Inputs that occured on the Player entity relative to the desync including those that are processed locally. For the Host Frames in question, the actions were exclusively the Player moving via NetworkPlayerMovementComponent.
+
+### Custom Event
+
+Custom Events are custom auditing events the developer can specify via macros.
+
+| Property | Description |
+|---|---|---|
+| AZ_MPAUDIT_INPUT_REWINDABLE | Allows the developer to audit a RewindableObject's current local value in comparison to its last known server value |
```

Comment:
Could we have an example usage of these in the code? Just a few line would work.